docker/lisk-service/env/gateway.env

```diff
@@ -13,7 +13,11 @@ HTTP_CACHE_CONTROL_DIRECTIVES=public,max-age=10
 SERVICE_GATEWAY_REDIS_VOLATILE=redis://redis_gateway_volatile:6379/1
 
 # Websocket rate limit config
-ENABLE_WS_RATELIMIT: false
-WS_RATELIMIT_POINTS: 5
-WS_RATELIMIT_DURATION: 1 # in seconds
+RATE_LIMIT_ENABLE_WS=false
+RATE_LIMIT_CONNECTIONS_WS=5
+RATE_LIMIT_DURATION_WS=1 # in seconds
 
+# HTTP rate limit config
+RATE_LIMIT_ENABLE_HTTP=false
+RATE_LIMIT_CONNECTIONS_HTTP=200
+RATE_LIMIT_WINDOW_HTTP=10 # in seconds
```

Comment:
Let us please update the environment variables to follow the pattern as suggested. This would bring in much more readability within the config file and would help us easily segregate them visually.

```suggestion
WS_RATE_LIMIT_ENABLE=false
WS_RATE_LIMIT_CONNECTIONS=5
WS_RATE_LIMIT_DURATION=1 # in seconds

# HTTP rate limit config
HTTP_RATE_LIMIT_ENABLE=false
HTTP_RATE_LIMIT_CONNECTIONS=200
HTTP_RATE_LIMIT_WINDOW=10 # in seconds
```